test/cartesian/ErrorBar.spec.tsx

```diff
@@ -103,4 +126,94 @@ describe('<ErrorBar />', () => {
 
     expect(container.querySelectorAll('.recharts-errorBar')).toHaveLength(10);
   });
+
+  test('Renders Error Bars with animation', async () => {
+    mockAnimation();
```

Comment:
afterthought and nit: but do the mock and the cleanup in a before/afterEach block here since you do it the same for each test

Reply:
good point! I've updated the code